include/vi.h

```diff
@@ -0,0 +1,240 @@
+/**
+ * @file vi.h
+ * @brief Video Interface Subsystem
+ * @ingroup display
+ */
+#ifndef __LIBDRAGON_VI_H
+#define __LIBDRAGON_VI_H
+
+#include <stdint.h>
+#include <stdbool.h>
+#include "regsinternal.h"
+#include "n64sys.h"
+
+/**
+ * @addtogroup display
+ * @{
+ */
+
+typedef uint32_t vi_register_t;
```

Comment:
Honestly I don't think this brings much, it seems to just hide the real type and it doesn't provide specific benefits I think. We use uint32_t* almost everywhere for registers anyway.